Sources/SwiftParser/StringLiterals.swift

```diff
@@ -487,7 +487,8 @@ extension Parser {
       // string literal.
       guard currentToken.leadingTriviaText.isEmpty else { break }
 
-      if let stringSegment = self.consume(if: .stringSegment) {
+      if self.at(.stringSegment, .identifier) {
+        let stringSegment = self.consumeAnyToken(remapping: .stringSegment)
```

Comment:
I think this can just be


```suggestion
      if let stringSegment = self.consume(if: .stringSegment, TokenSpec(.identifier, remapping: .stringSegment)) {
```